data/incipias/first contact.txt

```diff
@@ -0,0 +1,287 @@
+			`    When you arrive at the embassy you are a little bit exhausted from carrying it all the way and you are really grateful that the Hai remember you and directly guide you to the person you spoke to last time. "Hello Captain <first> <last>, nice to see you again and I see you are carrying something special with you, may I ask what this plant is?". "Of course," You say, "This is the prove you asked me to bring you." Now the Hai looks closer at the plant. "I see that this is an unusual plant but why is this a prove of friendship with another species?" You answer him, "This plant was genmanipulated to a degree that it can survive in nearly any climate, this is nothing humans could have done." The Hai looks at you, "You mind if we test this? We will make sure to not use to extreme methods, we do not want to kill it." You agree. The Hai gets two helpers to carry the plant and tells you to wait for his return.`
+			`    About an hour later the Hai returns with the plant and a excited looking face. "This plant is wonder, we exposed it to various environmental factors and some of them were really extreme, but all that happened was that the air got more breathable. This really is a worthy prove of a friendship with another species."`
+			label haihelp
+			`    Now the Hai turn to you with a very serious expression, "We have promised you to help you and now we will do so. Please come back in no earlier than four days to take a delegation of us to the Quarg ringworld. We need some time to prepare some stuff. You will meet us on Mirrorlake." You agree and say goodbye.`
+		event "Hai preparation time" 4
+
+event "Hai preparation time"
+
+mission "wanderer peace prove"
+	landing
+	name "Prove of Peace"
+	source
+		government "Wanderer"
+		not planet "Vara K'chrai"
+	destination "Vara K'chrai"
+	to offer
+		and
+			has "searching for a prove"
+			and
+				not "coalition peace prove: offered"
+				"reputation: Wanderer" >= 5
+	on offer
+		conversation
+			`Now that you are on a wanderer world you decide this would be a good time for asking them about the prove the Hai requested from you. You head for a near embassy.`
+			`    At the embassy an older Wanderer ask you what you are there for. You remember the Hai telling you that you should not tell the other species about the Incipias, so you search for something else to tell them.`
+			choice
+				"I am searching for something unique that presents your species best."
+					goto unique
+				"I am gathering proves of friendship from all over the galaxy."
+			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I got an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`
```

Comment:
```suggestion
			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I have an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`
```
The dialogue with the Wanderer should probably be adjusted to include the translation device's suggestions (for examples, look at the Wanderer missions).